### DISCUSSION TITLE
Tabby does not ask for a hostname

### DISCUSSION THREAD

**Discussion by KitLeskinen** [Eugeny/tabby#6661] [Q&A]
Hi!
I set as default ssh connection without hostname
When I press "+" button Tabby asks for username and tries to connect to blank hostname
How can I make the program ask for the hostname?

**Comment by Eugeny** [comment]
You can't use a profile without a hostname. Just use the "profile selector" button instead, where you can just type in a hostname or a `user@host:port`-style connection string.
